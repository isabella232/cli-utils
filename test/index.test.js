const { getOpts, getHelp, validate } = require('../lib/index');

const flagSchema = require('./fixtures/flags.json');

describe('index', () => {
  test('exports', () => {
    expect(getHelp).toBeDefined();
    expect(getOpts).toBeDefined();
    expect(validate).toBeDefined();
  });

  test('opts', () => {
    expect(getOpts(flagSchema)).toMatchSnapshot();
  });
});
