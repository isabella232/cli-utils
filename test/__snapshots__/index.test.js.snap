// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index opts 1`] = `
Object {
  "content": Object {
    "type": "string",
  },
  "dev-ware": Object {
    "type": "object",
  },
  "help": Object {},
  "port": Object {
    "type": Array [
      "number",
      "string",
    ],
  },
  "require": Object {
    "alias": "r",
    "type": "string",
  },
  "things": Object {
    "alias": "batman",
    "type": "array",
  },
  "version": Object {},
}
`;
